cmd/antrea-agent/agent_windows.go

```diff
@@ -0,0 +1,119 @@
+// +build windows
+
+// Copyright 2021 Antrea Authors
+// Copyright 2018 The Kubernetes Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
```

Comment:
please add a permalink to the original source code you copy-pasted

it may actually make more sense to copy https://github.com/kubernetes/kubernetes/tree/master/pkg/windows/service to our `third_party/` directory, even if it is just one file